src/test/skript/tests/syntaxes/effects/EffPlaySound.sk

```diff
@@ -0,0 +1,3 @@
+test "play sound effect":
+	play sound "block.stone.break" with volume 1 at spawn of world "world"
+	play sound "BLOCK_STONE_BREAK" with volume 1 at spawn of world "world"
```

Comment:
```suggestion
	play sound "block.stone.break" with volume 1 at test-location
	play sound "BLOCK_STONE_BREAK" with volume 1 at test-location
```

Reply:
tests seem to all be failing because of this, I will be reverting

Reply:
oh yeah this is patch my bad